roles/sap_hana_rhv_hypervisor/tasks/assert-rhv-hooks.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Check file permissions
```

Comment:
maybe it is better to use ansible.builtin.stat module to check file permissions?